index.html

```diff
@@ -5635,9 +5648,9 @@ <h3>Language and Base Direction</h3>
 
         <pre class="example nohighlight" title="Arabic text with a base direction of right-to-left">
 "title": {
-  "value": "<span class="highlight">HTML و CSS: تصميم و إنشاء مواقع الويب</span>",
-  "lang": "<code>ar</code>"
-  "dir": "<code>rtl</code>"
+  "@value": "<span class="highlight">HTML و CSS: تصميم و إنشاء مواقع الويب</span>",
```

Comment:
You want a `dir` on the span here so the string displays RTL correctly :-)

```suggestion
  "@value": "<span class="highlight" dir="rtl" lang="ar">HTML و CSS: تصميم و إنشاء مواقع الويب</span>",
```

Reply:
> You want a `dir` on the span here so the string displays RTL correctly :-)

Nope, that "highlight" is ReSpec markup on the example itself, not the markup that will be displayed in the spec. :)

We're highlighting the text that `@language` and `@direction` is being applied to... we're not using HTML to express `lang` and `dir`... and yes, it's difficult to see that just by staring at the source code in the spec. :)

Reply:
Not sure I understand. The `<span>` does end up in the rendered page (Respec is using it to style the string). By adding the dir and lang it makes the string display correctly (as if the @language and @direction had been applied):

![image](https://github.com/w3c/vc-data-model/assets/69082/16238011-c378-4b12-8e0f-a1198e804740)